### DISCUSSION TITLE
how do I change control-shift-v shortcut to control-v?

### DISCUSSION THREAD

**Discussion by paulpreibisch** [wavetermdev/waveterm#895] [General]
I'd like to change default keyboard shortcuts, like instead of control-shift-v for paste, just control-v. How can I do that?

**Comment by denisdamata** [comment]
I'm having the same problem, I want to edit the default keybindings/keyboard shortcuts. 

The block alternator <kbd>Alt</kbd>+<kbd>Shift</kbd>+<kbd>(1...9-0)</kbd> is very bureaucratic and I'd like to change it for something simpler.

**Comment by tkossak** [comment]
Any plans on introducing keybindings customization?
